packages/mongo-nav/src/project-nav/ProjectNav.tsx

```diff
@@ -59,7 +65,7 @@ const navContainerStyle = css`
 const menuIconButtonStyle = css`
   z-index: 1;
   ${facepaint({
-    marginRight: ['20px', '14px', '20px'],
+    marginRight: ['16px', '14px', '16px'],
```

Comment:
Wondering what this change is for?

Reply:
To better align the two badges, as they are in the design spec